.github/issue_template.md

```diff
@@ -0,0 +1,16 @@
+1. Please speak English, this is the language everybody of us can speak and write.
+2. Please ask questions or config/deploy problems on our pREST channel: https://gitter.im/nuveo/prest
+3. Please take a moment to search that an issue doesn't already exist.
+4. Please give all relevant information below for bug reports, incomplete details will be handled as an invalid report.
+
+**You MUST delete the content above including this line before posting, otherwise your issue will be invalid.**
+
+- pREST version (or commit ref):
+- PostgreSQL version:
+- Operating system:
+- Go version:
+- Log gist:
+
```

Comment:
we can ask for a URL too, because pREST use querystring to build where SQL clause